Public/New-HTMLChart.ps1

```diff
@@ -46,13 +46,14 @@
         if ($Setting.ObjectType -eq 'Bar') {
             # For Bar Charts
             if (-not $Type) {
-                # thiss makes sure type is not set if BarOptions is used which already set type to BarStacked or similar
+                # this makes sure type is not set if BarOptions is used which already set type to BarStacked or similar
                 $Type = $Setting.ObjectType
             }
             $DataSet.Add($Setting.Value)
             $DataName.Add($Setting.Name)
 
-        } elseif ($Setting.ObjectType -eq 'Pie' -or $Setting.ObjectType -eq 'Donut') {
+        }
+        elseif ($Setting.ObjectType -eq 'Pie' -or $Setting.ObjectType -eq 'Donut') {
```

Comment:
Can you fix those lines with open brackets on new line? I use OTBS code formatting. 

I guess it's my fault for not pushing my workspace. 
![image](https://user-images.githubusercontent.com/15063294/99297374-77f17000-2848-11eb-95ba-b46511d2af2d.png)

Reply:
Sure thing. Just did that. Let me know what else you need.